styles/_header.scss

```diff
@@ -72,42 +96,123 @@ $header-height: 71px;
       color: $link_color;
       text-decoration: underline;
     }
-
-    &.sign-in { display: inline-block; }
   }
+}
 
-  .hide-on-mobile {
-    @include mobile {
-      @include visually-hidden;
-    }
-  }
+.nav-wrapper-mobile {
+  @include desktop { display: none; }
 
-  .menu-button {
-    @include tablet { display: none; }
+  .menu-button-mobile {
     background: none;
     border: 0;
-    color: $link_color;
-    display: inline-block;
-    margin-right: 10px;
-    padding: 0;
     width: auto;
+    min-width: 71px;
```

Comment:
![image](https://user-images.githubusercontent.com/12835055/145229413-67b968ee-7a34-463f-b7f2-d73cb3511c31.png)

I wanted to avoid that on smaller screens.